cl/audio/templates/oral_argument.html

```diff
@@ -95,16 +95,20 @@ <h3 class="inline"><span>Your Notes</span></h3>
 
 {% block content %}
     <article class="col-sm-9">
-        <i id="favorites-star"
-           class="{% if favorite_form.instance.audio_id %}gold fa-star{% else %}gray fa-star-o bold{% endif %} pointer fa inline"
-           data-toggle="modal"
-           data-target="#modal-save-favorite, #modal-logged-out"
-           title="{% if favorite_form.instance.audio_id %}Edit this favorite{% else %}Save this as a favorite in your profile{% endif %}"></i>
+
 
         <h2 class="inline">{{ af|best_case_name|safe|v_wrapper }}</h2>
         {% include "includes/favorites_modal.html" %}
 
         <h3>{{ af.docket.court }}</h3>
+        <p>
+          <button id="favorites-star"
+           class="btn {% if favorite_form.instance.audio_id %}btn-danger{% else %}btn-success{% endif %} pointer hidden-print"
```

Comment:
I don't think it should be red when we have a favorite. Could we try playing with the color of the icon, maybe?

Reply:
I've changed the color to yellow (bootstrap `btn-warning`)
![Screen Shot 2023-01-17 at 14 54 57](https://user-images.githubusercontent.com/486004/213011052-79cb364c-d493-4edd-ba5c-9b787502e5c0.png)

Because only changing the icon color it's difficult to notice it:
![Screen Shot 2023-01-17 at 14 55 57](https://user-images.githubusercontent.com/486004/213011274-23309ebc-62fe-438a-9a01-f2195776c8ec.png)

Let me know if it seems good to you or I could try other combinations.